omod/src/main/webapp/configureServer.jsp

```diff
@@ -11,22 +13,26 @@
 <form method="POST">
   <table>
     <tbody>
-    <tr>
-      <td><spring:message code="dhisconnector.url"/></td>
-      <td><input name="url" type="text" size="30" value="${url}"/></td>
-    </tr>
-    <tr>
-      <td><spring:message code="dhisconnector.user"/></td>
-      <td><input name="user" type="text" size="20" value="${user}"/></td>
-    </tr>
-    <tr>
-      <td><spring:message code="dhisconnector.pass"/></td>
-      <td><input name="pass" placeholder="<hidden>" type="password" size="20"/></td>
-    </tr>
-    <tr>
-      <td/>
-      <td><input name="submit" type="submit" value="<spring:message code="dhisconnector.save" />"/></td>
-    </tr>
+    <openmrs:hasPrivilege privilege="View Connection,Manage Connection">
```

Comment:
thanks @Piumal1999  why not pu the privilege on the entire form?

Reply:
I'll update it. Thanks for the suggestion.

Reply:
fixed with 53c2484